### PR TITLE
feat(ingest/presto-on-hive): Extracting all the table properties from Hive Metastore 

### PR DESCRIPTION
**Pull request by treff7es**
Extracting all the table properties from Hive Metastore  and not just the description 

CleanShot 2023-06-30 at 15 52 41@2x

## Checklist

- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [ ] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

### CODE REVIEW

File: metadata-ingestion/src/datahub/ingestion/source/sql/presto_on_hive.py [hunk 1]
```diff
@@ -129,11 +129,6 @@ class PrestoOnHiveConfig(BasicSQLAlchemyConfig):
         description="Add the Presto catalog name (e.g. hive) to the generated dataset urns. `urn:li:dataset:(urn:li:dataPlatform:hive,hive.user.logging_events,PROD)` versus `urn:li:dataset:(urn:li:dataPlatform:hive,user.logging_events,PROD)`",
     )
 
-    extra_properties: List[str] = Field(
```
Comment (treff7es) [inline metadata-ingestion/src/datahub/ingestion/source/sql/presto_on_hive.py:132]: I removed this at all as nobody uses this property, and it didn't even work in the past.

File: metadata-ingestion/src/datahub/ingestion/source/sql/presto_on_hive.py [hunk 2]
```diff
@@ -488,22 +527,18 @@ def loop_tables(
...
 
             # add table properties
-            default_properties = ["create_date", "table_type", "table_location"]
-            properties: Dict[str, str] = {}
-            for prop in default_properties + self.config.extra_properties:
-                if prop in columns[-1]:
-                    properties[prop] = str(columns[-1][prop]) or ""
+            properties: Dict[str, str] = properties_cache.get(dataset_name, {})
+            properties["table_type"] = str(columns[-1]["table_type"]) or ""
+            properties["table_location"] = str(columns[-1]["table_location"]) or ""
```
Comment (mayurinehate) [inline metadata-ingestion/src/datahub/ingestion/source/sql/presto_on_hive.py:532]: I wonder if we should check the value of `columns[-1]["xxx"]` before stringification.
str(None) = 'None' (a truthy value)

File: metadata-ingestion/src/datahub/ingestion/source/sql/presto_on_hive.py [hunk 3]
```diff
@@ -488,22 +527,18 @@ def loop_tables(
...
                 [c["col_name"] for c in columns if c["is_partition_col"]]
             )
             if par_columns != "":
                 properties["partitioned_columns"] = par_columns
 
-            table_description = (
-                columns[-1]["description"] if "description" in columns[-1] else ""
-            )
-
+            table_description = properties.get("comment", "")
```
Comment (mayurinehate) [inline metadata-ingestion/src/datahub/ingestion/source/sql/presto_on_hive.py:541]:
```suggestion
            table_description = properties.get("comment")
```
If "comment" property is absent in properties, maybe we should not set table_description to "" , and instead keep it null.

**Review by mayurinehate — APPROVED** [review]
lgtm

**Review by hsheth2 — APPROVED** [review]

**Comment by hsheth2** [comment]
@treff7es looks like there's a bunch of conflicts on this PR

**Pull request merged**